index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Intuit Accounts - Sign Up</title>
+    <link rel="stylesheet" href="stylesheets/reset.css" />
+    <link rel="stylesheet" href="stylesheets/main.css" />
+  </head>
+  <body>
+    <header id="main-header" class="flex-h">
+      <div class="nav-container flex-h align-center">
+        <div class="m-r-1">
+          <p>Already have an Intuit account?</p>
+          <a id="forgot-id-pass" href="#">I forgot my User ID or Password</a>
+        </div>
+        <div class="m-r-1">
+          <button id="sign-in-button" class="btn">
+            <i class="fas fa-lock"></i>
+            Sign In
+          </button>
+        </div>
+      </div>
+    </header>
+    <!--End of Header Section-->
```

Comment:
Remove this comment and all other comments.👆

Reply:
The requested changes have been implemented.